e2e/test/scenarios/visualizations-tabular/object_detail.cy.spec.js

```diff
@@ -101,30 +101,28 @@ describe("scenarios > question > object details", { tags: "@slow" }, () => {
             alias: "Products",
           },
         ],
-        limit: 5,
+        limit: 2,
```

Comment:
We don't have a way to scope the search for the table cell within a single row (see this thread). We have to grab ALL table cells like this `cy.findAllByTestId("cell-data")`. That includes the header table cells btw.

So it makes sense to limit the number of total rows (the number of total cells) for the performance reasons.